sdk/eventhubs/azure_messaging_eventhubs/README.md

```diff
@@ -156,82 +109,96 @@ More information about Event Hubs features and terminology can be found here: [l
 
 # Examples
 
-Examples for various scenarios can be found on in the samples directory in our GitHub repo for
-[Event Hubs](https://github.com/Azure/azure-sdk-for-rust/tree/main/sdk/eventhubs/azure-messaging-eventhubs/samples).
+Additional examples for various scenarios can be found on in the examples directory in our GitHub repo for
+[Event Hubs](https://github.com/Azure/azure-sdk-for-rust/tree/main/sdk/eventhubs/azure_messaging_eventhubs/examples).
 
-## Open an Event Hubs message producer on an Event Hubs instance.
+<!-- no toc -->
+* [Open an Event Hubs message producer on an Event Hub instance](#open-an-event-hubs-message-producer-on-an-event-hub-instance)
+* [Send events](#send-events)
+  * [Send events directly to the Event Hub](#send-events-directly-to-the-event-hub)
+  * [Send events using a batch operation](#send-events-using-a-batch-operation)
+* [Open an Event Hubs message consumer on an Event Hubs instance](#open-an-event-hubs-message-consumer-on-an-event-hub-instance)
+* [Receive events](#receive-events)
```

Comment:
Did you check these all still work? In fact, come to think of it, I don't think I updated them for Key Vault secrets or keys. Could you double check?

Reply:
Just checked all the samples for Event Hubs, Secrets and Keys. 
They all compile fine. I ran `cargo fmt` on the secrets and keys samples.

Reply:
No, I mean that the TOC actually points to bookmarks that exist. I'll check them quick...

Reply:
Oh. Yes, I checked, they all work and point to the examples listed below.